<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeoContextHelpDialogBase</class>
 <widget class="QDialog" name="GeoContextHelpDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1091</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GeoContext - Documentation</string>
  </property>
  <widget class="QTabWidget" name="tabTables">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1071</width>
     <height>551</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::South</enum>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>1071</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QgsFileWidget" name="table_output_file">
      <property name="storageMode">
       <enum>QgsFileWidget::SaveFile</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnExport">
      <property name="text">
       <string>Export</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QComboBox" name="cbTableTabs"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
